patch-package
--- a/node_modules/expo-graphics/lib/ARCameraState.js
+++ b/node_modules/expo-graphics/lib/ARCameraState.js
@@ -13,8 +13,8 @@ const removeSuffix = str => {
 let TrackingStates;
 let TrackingStateReasons;
 if (AR.isAvailable()) {
-  TrackingStateReasons = AR.TrackingStateReasons;
-  TrackingStates = AR.TrackingStates;
+  TrackingStateReasons = AR.TrackingStateReason;
+  TrackingStates = AR.TrackingState;
 } else {
   TrackingStateReasons = {};
   TrackingStates = {};
--- a/node_modules/expo-graphics/lib/View/index.native.js
+++ b/node_modules/expo-graphics/lib/View/index.native.js
@@ -50,7 +50,7 @@ export default class GraphicsView extends React.Component<Props> {
     arRunningProps: {},
     arCameraProps: {},
     isShadowsEnabled: false,
-    arTrackingConfiguration: AR.TrackingConfigurations.World,
+    arTrackingConfiguration: AR.TrackingConfiguration.World,
     isArCameraStateEnabled: true,
   };
 
@@ -174,7 +174,7 @@ export default class GraphicsView extends React.Component<Props> {
 
     if (isArEnabled === true) {
       const trackingConfiguration =
-        arTrackingConfiguration || AR.TrackingConfigurations.World;
+        arTrackingConfiguration || AR.TrackingConfiguration.World;
       // Start AR session
       await AR.startAsync(
         findNodeHandle(this.nativeRef),
